packages/ia-activity-indicator/test/ia-activity-indicator.test.js

```diff
@@ -0,0 +1,34 @@
+import { html, fixture, expect } from '@open-wc/testing';
```

Comment:
**Suggestion**: replace the 'processing' and 'complete' strings with references to the appropriate `IAActivityIndicatorMode` property throughout.